ansible/roles/cli/tasks/copy_local_openwhisk_cli.yml

```diff
@@ -5,6 +5,20 @@
 - local_action: stat path={{ openwhisk_cli.local.location }}/{{ item }}
   register: binary_path
 
+- name: "ensure Nginx cli directory is writable"
+  file:
+    path: "{{ cli.nginxdir }}"
+    state: directory
+    mode: 0777
+  become: true
```

Comment:
do not hardcode `become` use variable `cli.dir.become`
```
name: "ensure Nginx cli directory is writable"
   file:
   path: "{{ cli.nginxdir }}"
   state: directory
   mode: 0777
   become: "{{ cli.dir.become }}"
```

Reply:
done